### ISSUE TITLE
Confidence level of candidates

### ISSUE THREAD

**Issue by subhash-velou** [bakwc/JamSpell#53]
I would like to know if there is a way to access confidence level/score for each candidate. For example, in the output below, can we know that the model is 90% confident it's "table", 4% confident its "tale" etc.?
`{
    "results": [
        {
            "candidates": [
                "table",
                "tale",
                "tacle",
                "tadle",
                "ladle",
                "tadre"
            ],
            "len": 5,
            "pos_from": 0
        }
    ]
}`

I'm having a requirement where I need to apply the fix with top candidate _only_ if the confidence level is above 90%. Would like to understand if you already have this facility or any plans to implement in future.

**Comment by hejazizo** [comment]
hey. Any luck with this? I exactly need this as sometimes the model makes mistake on task specific texts. Is this going to be implemented or is already implemented?

**Comment by hejazizo** [comment]
Anybody here?!!!

**Comment by jackneil** [comment]
i've created a mod that can output JSON with scores assigned to each candidate. Can't open a pull request here to push it though.

**Comment by bakwc** [comment]
why not? will be glad to accept it

**Comment by jackneil** [comment]
sorry i was trying to create a pull request on master and it was greyed out. need to branch first?

**Comment by bakwc** [comment]
yep

**Comment by jackneil** [comment]
ok i just branched locally to confidenceIntervals but when trying to publish the branch is says permission denied. using vscode.

**Comment by bakwc** [comment]
you need to publish it to local repository and then create pull request

**Comment by rprilepskiy** [comment]
@jackneil, how is it going with confidenceIntervals feature? Could you create pull request? 
Do you need any help?

**Comment by jackneil** [comment]
Yes we got it implemented and functional. I'll see if the dev on that project can get it pulled and merged. Best

-jack

Jack Neil
CEO / Hank AI, Inc.
jack.neil@hank.ai
(w) +1.828.490.1203
(m) +1.803.466.7551

Sent via Superhuman ( https://sprh.mn/?vip=jack.neil@hank.ai )
